gatsby/content/projects/bots/rustix.mdx

```diff
@@ -0,0 +1,26 @@
+---
+layout: projectimage
+title: rustix
+categories:
+ - bot
+description: A graph-based Matrix bot in Rust
+author: James
+language: Rust
+license: MIT
+repo: https://gitlab.com/jpypi/rustix
+thumbnail: /docs/projects/images/rust-logo-blk.svg
```

Comment:
Understandable. I added it originally since I noticed that maubot/{gitlab, python, github} bot pages all use respective project logos.

I've removed the log and reference to it!